debian/startup/kolibri.default

```diff
@@ -56,12 +56,22 @@ then
   done
 fi
 
-# Load user configuration
+# Load non-modular final configuration
 
 if [ -e /etc/kolibri/daemon.conf ]
 then
   . /etc/kolibri/daemon.conf
 fi
 
+
+# It should be possible to use "~/.kolibri" for a relative
```

Comment:
Could you ellaborate a bit more this sentence?
Why should `~/.kolibri`  be usable in a config file? I don't think it's a good idea, maybe that would explain the "various" reasons afterwards.

Reply:
It also ensures that someone who _assumes_ `~/` to work won't have to spend time resolving that it doesn't work :)

> Why should ~/.kolibri be usable in a config file?

Isn't it normal for HOME-relative paths to work in config files? Try `sudo grep -r "~\/" /etc`